cmd/promtool/tsdb_test.go

```diff
@@ -158,6 +159,7 @@ func TestTSDBDump(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
```

Comment:
you forgot to run each test case in parallel

Reply:
We can't run these in parallel, they use `os.Stdout` which can't be shared across testcases.